corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -43,6 +43,63 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
         $profilesTab.koApplyBindings(profileManager);
     }
 
+    var appReleaseLogsModel = function () {
+        let self = {};
+        self.releaseLogs = ko.observableArray();
+        self.fetchLimit = ko.observable();
+        self.totalItems = ko.observable();
+        self.fetchState = ko.observable();
+
+        self.onPaginationLoad = function () {
+            self.goToPage(1);
+        };
+
+        self.goToPage = function (page) {
+            if (self.fetchState() === 'pending') {
+                return false;
+            }
+            self.fetchState('pending');
+            $.ajax({
+                url: self.reverse("paginate_release_logs"),
+                dataType: 'json',
+                data: {
+                    page: page,
+                    limit: self.fetchLimit,
+                },
+                success: function (data) {
+                    self.releaseLogs(
+                        _.map(data.app_release_logs, function (log) {
+                            log.status = log.is_released ? "Released" : "Testing";
+                            log.timestamp = new Date(log.timestamp).toLocaleString();
```

Comment:
i think we would need to render time based on user timezone. There is a timezone set at user level and also at project level. Does this do that for the user's local timezone in Javascript itself?

Reply:
Yup, fixed this with time based on user and project timezone on the data returned itself. Removed JS logic as there is no need to change the timestamp string in JS now.